src/Tizen.NUI.Scene3D/src/internal/Interop/Interop.SceneView.cs

```diff
@@ -0,0 +1,76 @@
+            public static extern void AddCamera(global::System.Runtime.InteropServices.HandleRef sceneView, global::System.Runtime.InteropServices.HandleRef camera);
+
+            [global::System.Runtime.InteropServices.DllImport(Libraries.Scene3D, EntryPoint = "CSharp_Dali_SceneView_RemoveCamera")]
+            public static extern void RemoveCamera(global::System.Runtime.InteropServices.HandleRef sceneView, global::System.Runtime.InteropServices.HandleRef camera);
+
+            [global::System.Runtime.InteropServices.DllImport(Libraries.Scene3D, EntryPoint = "CSharp_Dali_SceneView_GetCameraCount")]
+            public static extern uint GetCameraCount(global::System.Runtime.InteropServices.HandleRef sceneView);
+
+            [global::System.Runtime.InteropServices.DllImport(Libraries.Scene3D, EntryPoint = "CSharp_Dali_SceneView_GetSelectedCamera")]
+            public static extern global::System.IntPtr GetSelectedCamera(global::System.Runtime.InteropServices.HandleRef sceneView);
+
+            [global::System.Runtime.InteropServices.DllImport(Libraries.Scene3D, EntryPoint = "CSharp_Dali_SceneView_GetCamera_0")]
+            public static extern global::System.IntPtr GetCamera(global::System.Runtime.InteropServices.HandleRef sceneView, uint index);
+
+            [global::System.Runtime.InteropServices.DllImport(Libraries.Scene3D, EntryPoint = "CSharp_Dali_SceneView_GetCamera_1")]
+            public static extern global::System.IntPtr GetCamera(global::System.Runtime.InteropServices.HandleRef sceneView, string name);
+
+            [global::System.Runtime.InteropServices.DllImport(Libraries.Scene3D, EntryPoint = "CSharp_Dali_SceneView_SelectCamera_0")]
+            public static extern void SelectCamera(global::System.Runtime.InteropServices.HandleRef sceneView, uint index);
+
+            [global::System.Runtime.InteropServices.DllImport(Libraries.Scene3D, EntryPoint = "CSharp_Dali_SceneView_SelectCamera_1")]
+            public static extern void SelectCamera(global::System.Runtime.InteropServices.HandleRef sceneView, string name);
+
+            [global::System.Runtime.InteropServices.DllImport(Libraries.Scene3D, EntryPoint = "CSharp_Dali_SceneView_SetImageBasedLightSource")]
+            public static extern void SetImageBasedLightSource(global::System.Runtime.InteropServices.HandleRef sceneView, string diffuseUrl, string specularUrl, float scaleFactor);
+
+            [global::System.Runtime.InteropServices.DllImport(Libraries.Scene3D, EntryPoint = "CSharp_Dali_SceneView_UseFramebuffer")]
+            public static extern void UseFramebuffer(global::System.Runtime.InteropServices.HandleRef sceneView, bool useFramebuffer);
+
+            [global::System.Runtime.InteropServices.DllImport(Libraries.Scene3D, EntryPoint = "CSharp_Dali_SceneView_IsUsingFramebuffer")]
+            public static extern bool IsUsingFramebuffer(global::System.Runtime.InteropServices.HandleRef sceneView);
```

Comment:
` [return: global::System.Runtime.InteropServices.MarshalAs(global::System.Runtime.InteropServices.UnmanagedType.U1)]`

Required here.

Reply:
Thank you!